config/kafka-instance-types-configuration.yaml

```diff
@@ -1,11 +1,18 @@
-# This configuration defines a list of supported Kafka instance types and their available sizes for KAS Fleet Manager. 
-# Modify this file to add additional Kafka instance types and sizes or change the default values listed below.
+# This configuration defines a list of supported Kafka instance types and their associated configuration for KAS Fleet Manager.
```

Comment:
This PR requires adding the new configuration values in the SaaS template before merging it. The reason for it is that in case the PR is merged before adding the contents to the SaaS file the new validations would fail.